CompuRacer_Core/src/maingui.py

```diff
@@ -0,0 +1,18 @@
+import os
+import sys
+
+from src.gui import RequestsGUI
+
+
+class MainGUI:
+    def __init__(self, racer):
+        super().__init__()
+        self.racer = racer
+
+        self.show_requests_gui(racer)
```

Comment:
Moet je hier niet 4 argumenten meegeven?

Reply:
Nee, dit is alleen het initialiseren van de MainGUI. Wordt verder niet zoveel mee gedaan. MainGUI.show_requests_gui wordt aangeroepen in command_processor.py